pkg/satellite/config.go

```diff
@@ -0,0 +1,74 @@
+package satellite
+
+import (
+	"context"
+
+	"github.com/graphql-go/graphql"
+	"google.golang.org/grpc"
+	"storj.io/storj/pkg/auth/grpcauth"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/provider"
+	agraphql "storj.io/storj/pkg/satellite/graphql"
```

Comment:
Rename the package to `satelliteql`, so you wouldn't have to use package aliases.